clients/big_query/lib/google_api/big_query/v2/api/jobs.ex

```diff
@@ -339,7 +354,9 @@ defmodule GoogleApi.BigQuery.V2.Api.Jobs do
     }
     %{}
     |> method(:get)
-    |> url("/bigquery/v2/projects/#{project_id}/jobs")
+    |> url("/bigquery/v2/projects/{projectId}/jobs", %{
+         "projectId" => URI.encode_www_form(project_id)
```

Comment:
This is a good change. Did we add url encoding for all our APIs?

Reply:
It will be there if we rebuild all APIs